bin/init.js

```diff
@@ -239,6 +242,11 @@ const getAllFiles = (dir) => {
  */
 const replaceFileContent = (files, chunksToReplace, newChunk) => {
 	files.forEach((file) => {
+
+		if(path.basename(file) === 'init.js' && (chunksToReplace === 'Blank theme' || chunksToReplace === 'underscore theme' || chunksToReplace === 'blankTheme')) {
```

Comment:
If we are not supposed to make any replacement in `init.js`, other checks are not needed. 
```suggestion
		if(path.basename(file) === 'init.js') {
```